cli/pkg/kctrl/cmd/app/init/file_util.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package init
+
+import (
+	"fmt"
+	"os"
+)
+
+// Check if file exists
+func IsFileExists(filePath string) (bool, error) {
+	_, err := os.Stat(filePath)
+	if err == nil {
+		return true, nil
+	} else if os.IsNotExist(err) {
+		return false, nil
+	} else {
+		return false, fmt.Errorf("failed to check for the existence of file. Error is: %s", err.Error())
```

Comment:
fmt.Errorf("failed to check for the existence of file. Error is: %s", err.Error())
=> 
fmt.Errorf("Checking file: %s", err)

Reply:
Adopted